src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpRequestParser.cs

```diff
@@ -194,6 +194,10 @@ private HttpVariableDeclarationNode ParseVariableDeclaration()
                     {
                         ConsumeCurrentTokenInto(node);
                     }
+                    else if(CurrentToken is { Text: "_" })
```

Comment:
Since these three branches perform the same action, you can combine them into this:

```csharp
if (CurrentToken is { Text: "@" } or { Kind: HttpTokenKind.Word } or { Text: "_" })
{
    ConsumeCurrentTokenInto(node);
}
```